trade-finance-manager-api/src/v1/controllers/deal-add-tfm-data/index.js

```diff
@@ -29,7 +30,7 @@ const addTfmDealData = async (deal, sessionPortalUser) => {
     },
   };
 
-  const updatedDeal = await api.updateDeal({ dealId, dealUpdate, sessionPortalUser });
+  const updatedDeal = await api.updateDeal({ dealId, dealUpdate, userInformation: generatePortalUserInformation(sessionPortalUser) });
```

Comment:
Have noticed this, and a couple other places should be
```suggestion
  const updatedDeal = await api.updateDeal({ dealId, dealUpdate, userInformation: generatePortalUserInformation(sessionPortalUser._id) });
```

Reply:
Will push after review